src/FlipCounter.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+import FlipCountercontainer from "./FlipCounterCONAINER";
+
+interface Props {}
+
+interface State {
+  x: number;
+  value: string;
+}
+
+class FlipCounter extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      x: 0,
+      value: ""
+    };
+  }
+  add = () => {
+    this.setState({ x: this.state.x + 1 });
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({ value: event.target.value });
+  };
+
+  handleSubmit = (event: React.FormEvent) => {
+    this.setState({ x: parseInt(this.state.value) });
+
+    event.preventDefault();
+  };
+
+  render() {
+    let mystring = String(this.state.x);
+    var arr = mystring.split("");
+    arr.reverse();
+
+    if (arr.length < 6) {
+      for (var i = 0; i < 5; i++) arr.push("0");
+    }
+
+    return (
+      <div>
+        <p>mystring</p>
+        <h1>
+          <table>
+            <tbody>
+              <tr>
+                <td id="main">{arr[5]}</td>
```

Comment:
There is still no array iteration here.

Imagine, that you have 100 items in the array. Do you want to write the code like this?

```
<td id="main">{arr[99]}</td>
..95 more...
<td id="main">{arr[3]}</td>
<td id="main">{arr[2]}</td>
<td id="main">{arr[1]}</td>
<td id="main">{arr[0]}</td>
```

I believe no. There is a very simple way to iterate through the array in React world.